File: service/recite-words/src/main/java/org/noodle/bean/recite/words/ReciteWordsRequest.java
package org.noodle.bean.recite.words;

import org.noodle.beans.NoodleRequest;

/**
 * 单词背诵记录中的单词列表获取服务
 * @author wanghy
 */
public class ReciteWordsRequest extends NoodleRequest {
    /**
     * 背诵记录
     */
    private Long reciteId;

    public Long getReciteId() {
        return reciteId;
    }

    public void setReciteId(Long reciteId) {
        this.reciteId = reciteId;
    }
}

File: client/ReciteWords/sdk/recite/ReciteWordsRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace sdk.recite.words
{
    /// <summary>
    /// 单词背诵记录中的单词列表获取服务
    /// </summary>
    public class ReciteWordsRequest : NoodleRequest<List<ReciteWordsDto>>
    {
        public long ReciteId { get; set; }

        public override string ApiMethodName => "recite.words";

        public override void Validate() { }
    }
}
